docs/alerts/monitors/create-monitor.md

```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
 
-<img src={useBaseUrl('img/alerts/monitors/edit-recovery-settings1.png')} alt="logs trigger recovery toggle.png" style={{border: '1px solid gray'}} width="700"/>
+**Example: Logs - Static - Missing Data**
 
-For example, when the alert is set to `> 10` the recovery would be set to `<= 10` when inferred. Sumo Logic automatically resolves the incident when the resolution condition is satisfied.
+<img src={useBaseUrl('img/alerts/monitors/logs-static-missing.png')} alt="logs-static-missing" style={{border: '1px solid gray'}} width="600" />
 
-`Recover automatically when result is <threshold type> <threshold> for the selected time period`.
-
-#### Configurable Resolution Window
-
-When configuring monitor trigger conditions, you can set a resolution window to quickly resolve alerts when the underlying issues are fixed. This controls how long a monitor will wait prior to resolving the alert, when the underlying issues was corrected. For example, if your monitor is evaluating the last 60 minutes, you can specify a resolution window of 15 minutes. Once 15 minutes has elapsed with your monitor resolution window continuously satisfied, the alert will resolve. <br/><img src={useBaseUrl('img/alerts/monitors/config-resolution-window-2.png')} alt="config-resolution-window" style={{border: '1px solid gray'}} width="700"/>
+`Alert when missing data within <time range>`
 
 | Parameter | Description |
 |:--|:--|
-| Threshold type | How you want the value compared. Select either **greater than**, **greater than or equal**, **less than or equal**, or **less than**. |
-| Threshold | The value against which the resolution will be evaluated. You can specify any valid numeric value. |
-| Occurrence Type	| The time condition you want for recovering the alert. Select either at any time within or at all times. Choose at all times if you want all the data points for the given metric to meet threshold conditions in a given time range, before recovering an alert. Alternatively, choose at any time within if you want to recover an alert when only a single data point meets the threshold condition for the given time range. |
+| `<time range>` | The duration of time to evaluate (values range from 5 minutes to 24 hours). |
 
-For Metrics monitors, you can choose to recover based on a single data point below the threshold, or all data points below the threshold.<br/><img src={useBaseUrl('img/alerts/monitors/metricsmonitor.png')} alt="Monitors" style={{border: '1px solid gray'}} width="600" />
+For recovery, Sumo Logic will automatically resolves the incident when the resolution condition is satisfied.
 
 #### Anomaly detection method
 
-**Example: Logs Trigger Type - Critical**
+**Example: Logs - Anomaly - Critical**
 
 <img src={useBaseUrl('img/alerts/monitors/monitor-anomaly-logs.png')} alt="Monitor anomaly logs" style={{border: '1px solid gray'}} width="600" />
 
```

Comment:
```suggestion

`Alert when anomaly count is at least <count> (max. 1) at any time within <time range>'

```

Reply:
You need to add this line under the screenshot o be consistent with the other examples.